CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+## Local Development
+
+To modify Angular environment variables for your local environment, copy `src/environments/environment.local.example.ts` to `src/environments/environment.example.ts` and update variables as needed.
+
+### via Angular server
+
+```
+npm install
+npm run start:local
+```
+
+The Angular server will start a watch loop and listen on [localhost:4200](http://localhost:4200).
+
+### via docker-compose
```

Comment:
Should add a note here to start the k3d server from `hobbyfarm/hobbyfarm`.

Reply:
added